components/LoginForm/LoginForm.js

```diff
@@ -30,16 +30,18 @@ LoginForm.propTypes = {
     email: string,
     password: string,
   }),
+  redirectFunc: func,
 };
 
 LoginForm.defaultProps = {
   initialValues: {
     email: '',
     password: '',
   },
+  redirectFunc: undefined,
```

Comment:
Let's set the default to `lodash/noop`

Reply:
Also, let's just name this the full variable name: `redirectFunction`

Reply:
will do